adflow/pyADflow.py

```diff
@@ -3660,7 +3681,7 @@ def _setAeroProblemData(self, aeroProblem, firstCall=False):
             self.adflow.bcdata.setbcdata(nameArray, dataArray, groupArray, 1)
 
         if not firstCall:
-            self.adflow.initializeflow.updatebcdataalllevels()
+            self.adflow.initializeflow.updatebcdataalllevels
```

Comment:
Why doesn't this need brackets? Is this some weird fortran thing I don't know about?

Reply:
It should have brackets.  I must've delted them accidentally and that's probably why the tests are all failing

Reply:
I think there was another instance of this in the PR too